.golangci.yml

```diff
@@ -0,0 +1,36 @@
+run:
+  deadline: 10m
+
+linters:
+  enable:
+    - contextcheck
+    - errcheck
+    - gocritic
+    - gofmt
+    - goimports
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    - misspell
+    - staticcheck
+    - unconvert
+  disable:
+    - unused
```

Comment:
Let's not disable this. Enabling this highlights a *bug* in the code where `staticDGX2VoltaPolicy` because it *should* actually be returned by `NewStaticDGX2Policy`.

/cc @klueska

Reply:
Created #27 to address this.